fedmsg_meta_fedora_infrastructure/tests/fmn.py

```diff
@@ -243,6 +243,35 @@ class TestConfirmationUpdateStatus(Base):
         }
     }
 
+class TestRuleUpdate(Base):
+    """ Messages of this type are published whenever **the status of a
+    confirmation changes** in the `Fedora Notifications
```

Comment:
Yeah, the text here should be revised to something that reflects the actual message being tested.  Something like ``a rule is updated`` or something similarly vague would be fine.  :)